src/test/java/testresults/GetTestResultsDeletionFlag.java

```diff
@@ -0,0 +1,55 @@
+package testresults;
+
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import steps.TestResultsSteps;
+
+@WithTags(
+        {
+                @WithTag(type = "TestResults", name = "All"),
+                @WithTag(type = "TestResults", name = "Positive"),
+                @WithTag(type = "Service", name = "One"),
```

Comment:
Do we need here the service tag "One"? Because we are not using it any more from what I understood.